members/CusM4b1cb3821543517908.yaml

```diff
@@ -3,127 +3,50 @@ contact_form:
   method: post
   action: ""
   steps:
-    - visit: "https://www.regulations.gov/comment?D=ED-2018-OCR-0064-0001"
-    - wait:
-        - value: 6
+    - visit: "https://ngpvan-freedom-tools.herokuapp.com/federalRegister"
     - find:
-        - selector: div.GIY1LSJGSC form
+        - selector: input#document_id
```

Comment:
Minor: Just so you know, the tag portion is not necessary, as the ID should uniquely identify an element (so just `#document_id` here would work). It might even be more performant to just use the ID (but the difference would beyond negligible here)